ui_framework/doc_site/src/views/table/controlled_table.js

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+
+import {
+  KuiToolBar,
+  KuiToolBarSearchBox,
+  KuiToolBarSection,
+  KuiButton,
+  KuiButtonIcon,
+  KuiTable,
+  KuiToolBarText,
+  KuiControlledTable,
+  KuiTableRowCellLiner,
+  KuiPager,
+  KuiTableRow,
+  KuiTableRowCell,
+  KuiTableRowCheckBoxCell,
+  KuiTableHeaderCell,
+  KuiToolBarFooterSection,
+  KuiToolBarFooter,
+  KuiTableHeaderCheckBoxCell,
+  KuiTableBody,
+  KuiTableHeader,
+} from '../../../../components';
+
+import {
+  LEFT_ALIGNMENT,
+  RIGHT_ALIGNMENT
+} from '../../../../services';
+
+// Arbitrarily use the 3rd index in the sample data as the unique identifier for handling checkbox toggling.
+const CELL_ID_INDEX = 3;
+
+export class ControlledTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedItems: []
```

Comment:
My suggestion above was just a way to implement the existing (admittedly non-production-ready) functionality using the React components, so that we can move forward without being hung up on working out the right implementation. 😄 

I think we could have a pretty good discussion about the best way to implement this kind of logic (and then encode it within a component or a service), but I'd rather not block this PR on it. So to me, it makes sense to remove it for now and then implement it after we've worked our way through that discussion.